math/OpenBLAS/files/patch-MacOSX-mktemp.diff

```diff
@@ -1,6 +1,17 @@
---- c_check~	2023-04-02 04:18:01.000000000 +0800
-+++ c_check	2023-04-03 13:42:05.000000000 +0800
-@@ -257,7 +257,7 @@
+diff --git c_check c_check
+index e8f90e1..d9adb23 100755
+--- c_check
++++ c_check
+@@ -204,7 +204,7 @@ esac
+ 
+ no_avx512=0
+ if [ "$architecture" = "x86" ] || [ "$architecture" = "x86_64" ]; then
+-    tmpd=`mktemp -d`
++    tmpd=`gmktemp -d`
```

Comment:
Can you add a URL at the top of the patchfile to the bug report about this problem? Upstream should be correcting this by using `mktemp` in a portable manner, rather than in a GNU-specific way. We don't want to maintain this patch forever.

Reply:
I haven't filed any issue, I simple adjusted the patch to the actual version.

Reply:
Backported to upstream